cassandra/src/main/java/se/tre/freki/storage/cassandra/functions/ResultSetToRowListFunction.java

```diff
@@ -0,0 +1,31 @@
+package se.tre.freki.storage.cassandra.functions;
+
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.google.common.base.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class ResultSetToRowListFunction implements Function<ResultSet, List<Row>> {
```

Comment:
Not sure if it's better but maube `MultipleRowLoggingFunction`? Can you add some javadoc in any case?